pkgs/build-support/bintools-wrapper/default.nix

```diff
@@ -192,7 +192,7 @@ stdenv.mkDerivation {
       unset ldInner
     '') + ''
 
-      for variant in ld.gold ld.bfd ld.lld; do
+      for variant in ld.gold ld.bfd ld.lld ld.mold; do
```

Comment:
```suggestion
      for variant in ld.*; do
```
Is it time to do this instead?

Reply:
It's time to do so! Doing this also makes convenience for more new linkers.

Reply:
OK then apply that that diff / squash history as you see fit!